service/rtc/sfu.go

```diff
@@ -484,7 +487,21 @@ func (s *Server) InitSession(cfg SessionConfig, closeCb func() error) error {
 
 			s.log.Debug("received screen sharing stream", mlog.String("streamID", streamID), mlog.String("sessionID", us.cfg.SessionID))
 
-			outScreenTrack, err := webrtc.NewTrackLocalStaticRTP(params.RTPCodecCapability, genTrackID(trackTypeScreen, us.cfg.SessionID), random.NewID(), webrtc.WithRTPStreamID(remoteTrack.RID()))
+			// To improve concurrency and support larger calls we create NumCPU output tracks and randomly distribute the receivers among them.
+			createOutScreenTracks := func(num int) ([]*webrtc.TrackLocalStaticRTP, error) {
+				outTracks := make([]*webrtc.TrackLocalStaticRTP, num)
```

Comment:
I'm surprised you didn't get a go linting error on this -- they usually prefer a `make(.., 0, num)` and then append.

Reply:
No I think I've had the linter say specifically for this case, it calls it a "c style" use of indexing. Afaik, appending after `make(..., 0, cap)` won't have a performance hit as long as you don't go over cap.

Reply:
Then I am sorry but that would be an absurd linter to have on :p